api/src/dev/server/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM alpine
```

Comment:
I don't draw a line between code that runs in a system, code used to build/configure a system, code used to help integrate with other systems, etc., since every part is responsible for providing the developer/user experience I want to achieve. I keep root-level items of a repo to a minimum, so I might have a few other things like docs and a central developer script. The goal is that a developer looking at the repo for the first time would naturally have a narrow focus. The `image-folder/Dockerfile` is a convention I stole from somewhere else. Very open to alternatives—I've never had a strong convention for what belongs in `src`.